scripts/utils.py

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/env python3
+# ***************************************************************************
+# * Authors:     Carlos Oscar S. Sorzano (coss@cnb.csic.es)
+# *              David Maluenda (dmaluenda@cnb.csic.es)
+# *              David Strelak (dstrelak@cnb.csic.es)
+# *
+# * This program is free software; you can redistribute it and/or modify
+# * it under the terms of the GNU General Public License as published by
+# * the Free Software Foundation; either version 2 of the License, or
+# * (at your option) any later version.
+# *
+# * This program is distributed in the hope that it will be useful,
+# * but WITHOUT ANY WARRANTY; without even the implied warranty of
+# * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# * GNU General Public License for more details.
+# *
+# * You should have received a copy of the GNU General Public License
+# * along with this program; if not, write to the Free Software
+# * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
+# * 02111-1307  USA
+# *
+# *  All comments concerning this program package may be sent to the
+# *  e-mail address 'scipion@cnb.csic.es'
+# ***************************************************************************/
+
+import subprocess
+from os import environ, path, remove
+import distutils.spawn
+import glob
+
+
+def green(text):
+    return "\033[92m "+text+"\033[0m"
```

Comment:
I know this is a extremely dummy request. But, a reference to the color code would be very useful. Once it took me some time to figure out the code